index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>RESTAURANT-MENU</title>
+  <link rel="stylesheet" href="style.css">
+  <link=
+    href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Lato:ital,wght@0,300;0,700;1,700&display=swap"
+    rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+</head>
+
+<body>
+  <nav>
+    <div class="background">
+
+
+      <div class="first-bar">
+        <ul class="first-list">
+          <li><img src="images/facebook.svg" alt=""></li>
+          <li><img src="images/twitter.svg" alt=""></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+
+      <div class="bottom-nav">
+        <div class="top-nav">
+          <div id="harmburger">
+            <img src="images/menu.svg" alt="menu">
+          </div>
+        </div>
+      </div>
+
+      <div class="nav-bottom" id="menu">
+        <a href="index.html" class="nav-logo">
+          <span class="my-logo">
+            <img src="/images/main-logo.jpg" alt="mylogo">
+          </span>
+        </a>
+        <div class="close" id="cancel-btn">x</div>
+        <ul class="nav-menu">
+          <li> <a href="about.html">About</a></li>
+          <li> <a href="#">Program</a></li>
+          <li> <a href="#">News</a></li>
+          <li> <a href="#">Sponsor</a></li>
+          <li> <a class="campaign-item" href="#" id="button">CC Campaign</a></li>
+        </ul>
+      </div>
+
+
+    </div>
+    <!-- <div class="logo"><img src="./images/Restaurant-logo.jpg" alt=""></div> -->
+  </nav>
+  <header>
+    <h3 class="open-comments">HEY YOU THERE COME TASTE SOME!</h3>
+    <h1 class="welcome">WELCOME TO CHEF YOUNG'S RESTAURANT AKA THE MOST FAMOUS AND SUMPTUOUS RESTAURANT IN THE WORLD
+      OF 2023</h1>
+    <div class="paragraph">
+      <p>It is often said "The road to a man's heart is through his stomach", We warmly welcome you to our beautiful
+        and inviting restaurant, where we offer satisfying and quality food. A very spacious place for your lovely
+        Dates. </p>
+    </div>
+    <div class="location">
+      <h1>2023.12.01(FRI) ~ 31(SAT)</h1>
+      <p>@Gaius street near the Fancy park, Douala, Cameroon</p>
+    </div>
+  </header>
+  </div>
+
+  <section class="vicinity">
+    <h3 class="h3-vicinity">Vicinity</h3>
+    <div class="line"></div>
+
+    <div class="cards">
```

Comment:
- [ ] As you have tried to make changes to the card as suggested by the previous reviewer, I can still still that the alignments of each card item is still not matching the design guideline. Please take note that each element of each card takes a fixed width which is consistent across all the cards. KIndly fix this 👍🏽 

your design 👇🏽 
![image](https://github.com/Young-Gaius/capstone-project/assets/63473584/353f06c3-3ff5-4712-b3f4-3f3e90fbbc22)

design guideline 👇🏽 
![image](https://github.com/Young-Gaius/capstone-project/assets/63473584/b12f2f10-bdbb-4479-97a7-083ab8d727d9)